linux/tima/README-directory-size.md

```diff
@@ -0,0 +1,18 @@
+# directory size task
+
+#### Task:
+>To make a new directory and check it's size. Explain why does empty directory has a size.
+
+---
+
+>After making an empty directory we noticed that it's size is 4096 bytes.    
+>The 4096 bytes size for directories is often chosen, because it aligns with the block size of the file system. File systems are typically divided into fixed-size blocks, and the directory structure is designed to be efficient in terms of block allocation.
+>This means that even if directory contains only few entries, it still occupies en entire block, which is usually 4096 bytes in size. This simplifies file system managment and reduce fragmentation.
+>If you want to find the real storage of directory, you can use command `du` which stands for 'disk usage'
```

Comment:
```suggestion
After making an empty directory we noticed that it's size is 4096 bytes.    
The 4096 bytes size for directories is often chosen, because it aligns with the block size of the file system. File systems are typically divided into fixed-size blocks, and the directory structure is designed to be efficient in terms of block allocation.
This means that even if directory contains only few entries, it still occupies en entire block, which is usually 4096 bytes in size. This simplifies file system managment and reduce fragmentation.
If you want to find the real storage of directory, you can use command `du` which stands for 'disk usage'
```